engine/apps/webhooks/utils.py

```diff
@@ -184,4 +185,19 @@ def serialize_event(event, alert_group, user, webhook, responses=None):
         data["webhook"] = {"id": webhook.public_primary_key, "name": webhook.name, "labels": get_labels_dict(webhook)}
         data["integration"]["labels"] = get_labels_dict(alert_group.channel)
         data["alert_group"]["labels"] = get_alert_group_labels_dict(alert_group)
+
+    # Add additional webhook data if the integration has it
+    source_alert_receive_channel = webhook.filtered_integrations.filter(
+        connected_alert_receive_channels__isnull=False
+    ).first()  # TODO: is it possible to have more than one?
+    if source_alert_receive_channel and hasattr(source_alert_receive_channel.config, "additional_webhook_data"):
+        data.update(source_alert_receive_channel.config.additional_webhook_data(source_alert_receive_channel))
```

Comment:
Should we add this and the external ID under some key? (if we eventually have more than one external ID?)

Reply:
the idea is that `external ID` and `settings` are per "source" integration, so if there are multiple external IDs (e.g. 2 SNOW integrations) they would be referenced in two different sets of webhooks. not sure we want to expose all the different external IDs at this point, wdyt?

Reply:
hm.. right, I see, makes sense :+1: